x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/data_input_flyout/steps/rules/sub_steps/migration_name/index.tsx

```diff
@@ -22,18 +22,25 @@ export const useMigrationNameStep = ({
   setMigrationName,
   migrationName: storedMigrationName,
 }: MigrationNameStepProps): EuiStepProps => {
-  const { data: currentUserProfile } = useGetCurrentUserProfile();
+  const { data: currentUserProfile, isLoading } = useGetCurrentUserProfile();
 
   const migrationName = useMemo(() => {
     if (storedMigrationName) {
       return storedMigrationName;
     }
+    if (isLoading) {
+      return undefined; // profile is still loading
+    }
+
+    // localized date and time (e.g., "Wed, 01 Jan 2025 12:00:00 PM")
+    const datetime = moment(Date.now()).format('dddd, D MMM YYYY, h:mm:ss A');
+
     if (currentUserProfile?.user.username) {
```

Comment:
Please, consider using the full_name if it's available, so we fix [this one](https://github.com/elastic/security-team/issues/13380) as well:
```suggestion
    const name = currentUserProfile?.user.full_name || currentUserProfile?.user.username
    if (name) {
```

Reply:
I have pushed this change. but do you know how can we replicate it?